explorer-api/swagger.yaml

```diff
@@ -4,7 +4,7 @@ host: localhost
 basePath: /
 info:
   title: cardano-explorer-api
-  version: 2.1.0
+  version: 3.0.1
```

Comment:
I tried to make a diff of this on bump.sh, but who had guessed... this version of the yaml also fails to parse. After converting to json and uploading, the deployment errors :sweat_smile:

Reply:
That one was actually a _truly_ malformed OpenAPI 2.0 schema :sweat_smile:  .... I fixed it in: https://github.com/input-output-hk/cardano-rest/pull/106/commits/043a9c86842dd6951821a7337e49b8d6e0849790

Reply:
I migrated this to openapi 3.0.0 https://gist.github.com/hasufell/3ef235dce2fd303388c6aece21b6830a